### DISCUSSION TITLE
sqlc plugin

### DISCUSSION THREAD

**Discussion by delaneyj** [zombiezen/go-sqlite#107] [Ideas]
I've started work on a [sqlc](https://sqlc.dev/) plugin [here](https://github.com/delaneyj/toolbelt/tree/main/sqlc-gen-zombiezen).  Its trying to include niceties like converting time.Duration|Time and adding to as I need.  About to make the `Prep` separate from the execute but still enough is in place to start playing with.

**Comment by zombiezen** [comment]
Nice! I'm sure this will be helpful for other folks. Let me know if there's support from the library side that can help.
